src/igvfd/tests/test_upgrade_phenotypic_feature.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+def test_phenotypic_feature_upgrade_3_4(upgrader, phenotypic_feature_v1):
```

Comment:
test_phenotypic_feature_upgrade_3_4 should be test_phenotypic_feature_upgrade_1_2